### PR TITLE
Monica Parrillo  - Fatima's Pod

### PR DESCRIPTION
**Pull request by monicaleep**
Resubmit with filename changed. 
Felt pretty comfortable with this deliverable.

### CODE REVIEW

File: MonicaSolution.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+
+//2.  Excited Kitten
+const sayings = ["...human...why you taking pictures of me?...","..the catnip made me do it...", "meow?","...why does the red dot always get away..."];
+for(let i=0;i<10;i++){
+  console.log("Love me, pet me! HSSSSSS!")
+  if(i%2===0){
+    console.log(sayings[Math.floor(Math.random()*4)])
+  }
+
```
Comment (ghost) [inline MonicaSolution.js:15]: missing a closing curly bracket for the for loop!

File: MonicaSolution.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+console.log(`The current temp is ${currentTemp}F`)
+while(currentTemp!==desiredTemp){
+  if (currentTemp>desiredTemp){
+    currentTemp--
+    console.log(`The current temp is ${currentTemp}F`)
+  } else{
+    currentTemp++;
+    console.log(`The current temp is ${currentTemp}F`);
+  }
+}
```
Comment (ghost) [inline MonicaSolution.js:28]: solid indentation in this loop 💯

**Comment by ghost** [comment]
nice work! marking complete ✅

**Comment by monicaleep** [comment]
Thanks Fatima!

**Pull request closed without merging**